<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QWidget" name="Logowanie">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>150</y>
        <width>321</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Login"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Haslo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Haslo">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonZaloguj">
         <property name="text">
          <string>Zaloguj</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="PoZalogowaniu">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>251</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_PoZalAktuDostaw">
         <property name="text">
          <string>Aktualizacja dostawcow</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_PoZalAktuOferty">
         <property name="text">
          <string>Aktualizacja oferty</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_PoZalAktuStatDost">
         <property name="text">
          <string>Aktualizacja statusu dostawy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_PoZalRap1">
         <property name="text">
          <string>Raport1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_PoZalRap2">
         <property name="text">
          <string>Raport2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_PoZalWroc">
         <property name="text">
          <string>Wroc</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="AktStatus">
     <widget class="QTableView" name="tableViewAktStatus">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>470</y>
        <width>368</width>
        <height>69</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Id dostawy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_AktStatusId"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Status dostawy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_AktStatusStatus"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPushButton" name="pushButton_AktStatusAktu">
           <property name="text">
            <string>Aktualizuj</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_AktStatusWroc">
           <property name="text">
            <string>Wroc</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="AktDost">
     <widget class="QTableView" name="tableView_AktDost">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>370</y>
        <width>531</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Id</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Nazwa</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Email</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLineEdit" name="lineEdit_AktDosId"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_AktDostNazwa"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_AktDostEmail"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Miasto</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Ulica</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Nr ulicy</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLineEdit" name="lineEdit_AktDostMiasto"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_AktDostUlica"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_AktDostNrUlicy"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QPushButton" name="pushButton_AktDostDod">
           <property name="text">
            <string>Dodaj</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_AktDostAktua">
           <property name="text">
            <string>Aktualizuj</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_AktDostUsun">
           <property name="text">
            <string>Usun</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_AktDostWroc">
           <property name="text">
            <string>Wroc</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="AktOfert">
     <widget class="QTableView" name="tableView_AktOfert">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>1</y>
        <width>800</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>531</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Kod Oferty</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Cena za jednostke</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Cena za sztuke</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <item>
              <widget class="QLineEdit" name="lineEdit_AktOfertKod"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_AktOfertCenaJed"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_AktOfertCenaSzt"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Czas realizacji w dniach</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Id dostawcy</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_14">
             <item>
              <widget class="QLineEdit" name="lineEdit_AktOfertCzas"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_AktOfertIdDostawcy"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QPushButton" name="pushButton_AktOfertDodaj">
           <property name="text">
            <string>Dodaj</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_AktOfertAktu">
           <property name="text">
            <string>Aktualizuj</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_AktOfertUsun">
           <property name="text">
            <string>Usun</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_AktOfertWroc">
           <property name="text">
            <string>Wroc</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Rap1">
     <widget class="QPushButton" name="pushButton_Rap1Wroc">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>510</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Wroc</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_Rap1">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>1</y>
        <width>800</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Rap2">
     <widget class="QTableView" name="tableView_Rap2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>1</y>
        <width>800</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Rap2Wroc">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>510</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Wroc</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
